packages/shared-ux/chrome/navigation/src/ui/v2/use_register_tree_node.ts

```diff
@@ -17,8 +17,8 @@ import { useNavigationGroup } from './components/navigation_group';
  */
 export const useRegisterTreeNode = () => {
```

Comment:
```
v2
|- hooks
   |- index.ts        <- just export "use_init_navnode"
   |- use_init_navnode.ts
   |- use_register_tree_node.ts
```

something like that would make it easier for a person like me to see how everything is organized

Reply:
Addressed in https://github.com/elastic/kibana/pull/158297/commits/8a258d70f4733d61abc5405b9570594e85e69381